overrides/default.nix

```diff
@@ -973,11 +973,25 @@ lib.composeManyExtensions [
 
       mypy = super.mypy.overridePythonAttrs (
         old: {
-          MYPY_USE_MYPYC =
-            # is64bit: unfortunately the build would exhaust all possible memory on i686-linux.
-            stdenv.buildPlatform.is64bit
-            # Derivation fails to build since v0.900 if mypyc is enabled.
-            && lib.strings.versionOlder old.version "0.900";
+          patches = (old.patches or [ ]) ++ [
+            # FIXME: Remove patch after upstream has decided the proper solution.
+            #        https://github.com/python/mypy/pull/11143
+            (pkgs.fetchpatch {
+              url = "https://github.com/python/mypy/commit/f1755259d54330cd087cae763cd5bbbff26e3e8a.patch";
+              sha256 = "sha256-5gPahX2X6+/qUaqDQIGJGvh9lQ2EDtks2cpQutgbOHk=";
+            })
+          ];
+          buildInputs = old.buildInputs ++ [
```

Comment:
The safe way to write this would be `old.buildInputs or [ ] ++ [`.

Reply:
fixed